src/PlayerPhysicsComponent.cpp

```diff
@@ -19,4 +19,17 @@ void PlayerPhysicsComponent::updatePhysics(float deltaTime) {
       holder_->setPosition(newPosition);
       updateBoundingBox();
    }
+
+   // Check player collision against static objects
+   GameWorld& world = GameManager::instance().getGameWorld();
+   GameObjectType objTypeHit = world.checkCollision(holder_)->type;
+
+   if (objTypeHit == GameObjectType::STATIC_OBJECT) {
+      if (holder_->velocity != 0.0f) {
+         glm::vec3 newPosition = holder_->getPosition() - (holder_->velocity * 
```

Comment:
Yeah, right now it's just doing a check to see if it's colliding at all and moves the player backward.  I added it in just for initial testing; definitely want it based off of movement and how the player is hitting the object as suggested.